### DISCUSSION TITLE
Is it possible to replace rg / ripgrep with rga / ripgrep-all ?

### DISCUSSION THREAD

**Discussion by onjour** [BurntSushi/ripgrep#2816] [Q&A] [answered]
I'm not very tech savvy and I work with a lot of different files — including many PDFs.

I love _yazi_ and one thing I'm missing so far is the ability to search inside PDFs for text. 

I found that a [modified _rg_ exists](https://github.com/phiresky/ripgrep-all) specifically for this — but also many other file formats E-Books, Office documents, zip, tar.gz — and I've been wondering if it would be possible to replace the _rg_ search bound to the 'S' key in yazi to the _rga_ command that would better suit my needs? 

That would be great! If not, can someone help me to achieve PDF searching capability with the tools offered by default?

Thank you!

**Comment by BurntSushi** [comment, marked as answer]
Huh? This is ripgrep's issue tracker. It seems like you're looking for help configuring something called "yazi" (which I have never heard of).

**Reply by onjour** [reply]
Hi, yes I'm sorry I mixed up the repos and I couldn't delete my discussion.

Thank you for your time and your hard work!
